unikernel.ml

```diff
@@ -127,35 +182,33 @@ struct
     let resp = Httpaf.Response.create ~headers status in
     Httpaf.Reqd.respond_with_string reqd resp data
 
-  let http_response reqd ?(header_list = []) ~title ~data http_status =
-    let code = Httpaf.Status.to_code http_status
-    and success = Httpaf.Status.is_successful http_status in
-    let status = { Utils.Status.code; title; data; success } in
-    Lwt.return
-      (reply reqd ~content_type:"application/json" ~header_list
-         (Utils.Status.to_json status)
-         http_status)
-
   let sign_up reqd =
-    match Middleware.has_session_cookie reqd with
+    let now = Ptime.v (P.now_d_ps ()) in
+    let csrf = Middleware.get_csrf now () in
+    let csrf_cookie = csrf.name ^ "=" ^ csrf.value ^ ";Path=/;HttpOnly=true" in
+    match Middleware.has_cookie "molly_session" reqd with
```

Comment:
to avoid having to hardcode `"molly_session"` all over, maybe provide a `Middleware.has_session_cookie reqd`?

Reply:
done in 88895ec